digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -96,35 +151,27 @@ export default {
             this.isLoaded = true;
           });
       }
+      this.$socket.emit('joinRoom', this.$store.getters.getSessionId);
     },
 
     closeSession() {
       $cookies.set("SessionId", -1, "1h");
       this.isLoaded = false;
     },
+
+    toggleForm1() {
```

Comment:
Can you rename these 2 functions to something like `ToggleCreateForm` and `ToggleJoinForm`?